packages/web-app-files/tests/unit/views/FilesDrop.spec.js

```diff
@@ -13,7 +13,7 @@ localVue.use(GetTextPlugin, {
 localVue.prototype.$client.publicFiles = {
   PUBLIC_LINK_SHARE_OWNER: 'admin',
   // function is mocked because it should return a promise with a list of resources
-  list: async () => [{ getProperty: jest.fn((val) => val) }],
```

Comment:
Sadly this needs to stay to keep the unit test passing 🙄

Reply:
dang. will revert the `async` removals